### DISCUSSION TITLE
矽谷「失敗博物館」用「翻車」證明失敗未必是成功之母

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2076] [General]
https://technews.tw/2024/09/01/sean-jacobsohn-failure-museum/

一般人都會覺得「失敗」就是差勁，不敢輕易向他人提起，似乎只要表明了自己是失敗的，就永遠低人一等。
但館長 Sean Jacobsohn的看法完全相反，「如果創業者從沒嘗過失敗的滋味，代表他們沒有承擔過有意義的風險」，「如果你只談成功卻不聊失敗，就像數學等式少了一邊」，這兩句話是我看完這篇文章後印象最深的。這也讓我想到國中會考時的作文題目:未成功的物品展覽會。
其實「失敗」並非不好的，就像許多厲害的大公司老闆，也都是經歷過創業、投資、錯誤抉擇、經營管理等方面上的失敗，才有現在的成功；就像高中的學習歷程或任何專題報告，老師、教授想看到的不全然是我們成功且完美的執行計畫，失敗、有缺陷也沒關係，只要我們有從中反省、檢討，思考未來該如何改善、下一步還能做什麼，能看到我們學習與進步的過程，就是成長的代表。
而不論是科技公司的CEO、高中生或大學生，是經營企業、社團，還是寫專題報告，克服困難、從失敗中檢討的能力，是每個人在每個人生階段必備的重要養分。

**Comment by ds0w0** [comment]
失敗就代表曾經冒險嘗試過，而透過失敗的經驗我們才能認知自己能夠改進的地方。我覺得文章中把失敗紀錄下來的失敗博物館很有趣，不僅記錄了許多產品的更迭，也顯示了諸多創業的過程和血淚，這些足跡是會隨著路越走越遠而成為墊腳石的。

**Comment by lai1008** [comment]
我覺得每一次的失敗都會是下一次嘗試的很好養分，會知道自己那個環節出錯了，並且可以針對那個部分去做修正，然後經過多次的嘗試，最終也可以成功，但前提是要夠有耐心且有辦法接受多次的失敗。

**Comment by 113607514** [comment]
我認為每一次的失敗都是下一次嘗試的寶貴養分，因為它能讓我們明確知道自己在哪個環節出錯，並能針對那部分進行調整。經過反覆的嘗試，最終一定能夠成功，但前提是必須具備足夠的耐心，並能夠接受多次的失敗。
